doc/tutorials/pre-launched-rt.rst

```diff
@@ -30,60 +30,60 @@ Example of Pre-Launched RT
 
 Take the Whiskey Lake WHL-IPC-I5 board (as described in :ref:`hardware`) for
 example. This platform can connect both an NVMe and a SATA drive and has
-two Ethernet ports. We will passthrough the SATA and Ethernet 03:00.0
-devices into the Pre-Launched RT VM, and give the rest of the devices to
+two Ethernet ports. We will pass through the SATA and Ethernet 03:00.0
+devices into the pre-launched RTVM, and give the rest of the devices to
 the Service VM.
 
-Install SOS With Grub on NVMe
-=============================
+Install Service VM OS With GRUB on NVMe
+=======================================
 
-As with the Hybrid and Logical Partition scenarios, the Pre-Launched RT
+As with the Hybrid and Partition scenarios, the Pre-Launched RT
 mode must boot using GRUB.  The ACRN hypervisor is loaded as a GRUB
-multiboot kernel, while the Pre-Launched RT kernel and Service VM
-kernels are loaded as multiboot modules. The ACRN hypervisor, Service
-VM, and Pre-Launched RT kernel images are all located on the NVMe drive.
+multiboot kernel, while the pre-launched RTVM kernel and Service VM
+kernel are loaded as multiboot modules. The ACRN hypervisor, Service
+VM, and pre-launched RTVM kernel images are all located on the NVMe drive.
 We recommend installing Ubuntu on the NVMe drive as the Service VM OS,
 which also has the required GRUB image to launch Pre-Launched RT mode.
-Refer to :ref:`gsg`, to
-install Ubuntu on the NVMe drive, and use grub to launch the Service VM.
+Refer to :ref:`gsg` to
+install Ubuntu on the NVMe drive, and use GRUB to launch the Service VM.
 
 Install Pre-Launched RT Filesystem on SATA and Kernel Image on NVMe
 ===================================================================
 
-Follow the :ref:`gsg` to install RT rootfs on SATA drive.
+Follow the :ref:`gsg` to install RT rootfs on the SATA drive.
```

Comment:
Does GSG show how to do this?

Reply:
Not in the new version... looks like we need a new tutorial for this.

Reply:
> Not in the new version... looks like we need a new tutorial for this.

@gvancuts @dbkinder Should we postpone updating this file altogether until we can update the whole thing (like we chose to do for the Zephyr tutorial)? If so, I can open as issue and close this PR.

Reply:
Could we not write those now and keep this open until it's done? We can re-use a lot of the content from:
1. Install Ubuntu on the SATA disk: https://projectacrn.github.io/2.5/getting-started/getting-started.html#install-the-ubuntu-user-vm-rtvm-on-the-sata-disk
2. Build the Preempt-RT kernel (dev machine): https://projectacrn.github.io/2.5/getting-started/getting-started.html#build-and-install-the-rt-kernel-for-the-ubuntu-user-vm
3. Install the Preempt-RT modules on the SATA rootfs
4. Install the Preempt-RT kernel in the rootfs on the NVMe

Reply:
@gvancuts I don't have enough knowledge to make the changes myself. Can you update the document or do we need someone from the Eng team?

Reply:
> @gvancuts I don't have enough knowledge to make the changes myself. Can you update the document or do we need someone from the Eng team?

I could do it, just need to find the time for it. Likely early next week at this stage.

Reply:
@gvancuts Nanlin asked whether she and David can start cherry-picking merged PRs for the 2.7 release soon ... My suggestion: Can you submit your changes by your Thursday; otherwise, wait until after the 2.7 release?

Reply:
Update: @gvancuts will fix in a future release. We can merge for 2.7.